### PR TITLE
Use watch notification

### LINKED ISSUE #228: Use collection watch to publish notification

### PR DESCRIPTION
**Pull request by erdenezul**
Closes #228

### CODE REVIEW

**Comment by erdenezul** [comment]
done

File: src/db/listener.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      }
+    });
+
+    Conversations.watch().on('change', data => {
+      const conversation = data.fullDocument;
+
+      if (data.operationType === 'insert' && conversation) {
+        Customers.findOne({
+          _id: conversation.customerId,
+        }).then(doc => {
```
Comment (batamar) [inline src/db/listener.ts]: Can we use await ?

Comment (erdenezul) [reply src/db/listener.ts]: done

**Comment by erdenezul** [comment]
@batamar js heap error

**Pull request merged**